lectures/short_path.md

```diff
@@ -23,6 +23,16 @@ kernelspec:
 ```{index} single: Dynamic Programming; Shortest Paths
 ```
 
+```{contents} Contents
```

Comment:
@HumphreyYang we should remove these `content` blocks as they are not required. The theme builds a RHS toc in the margin.